drape/drape_tests/glyph_mng_tests.cpp

```diff
@@ -5,30 +5,51 @@
 #include "drape/bidi.hpp"
 #include "drape/font_constants.hpp"
 #include "drape/glyph_manager.hpp"
+#include "drape/text_itemizer.hpp"
+
+#include "qt_tstfrm/test_main_loop.hpp"
 
 #include "platform/platform.hpp"
 
 #include <QtGui/QPainter>
 
-#include "qt_tstfrm/test_main_loop.hpp"
-
-#include <cstring>
+#include <ft2build.h>
 #include <functional>
 #include <iostream>
-#include <memory>
 #include <vector>
+#include FT_FREETYPE_H
+#include FT_MODULE_H
+#include <harfbuzz/hb-ft.h>
+#include <harfbuzz/hb.h>
 
-using namespace std::placeholders;
-
-namespace
+namespace glyph_mng_tests
+{
+std::ostream& operator<<(std::ostream & stream, const dp::text::TextMetrics & tm)
```

Comment:
nit: We usually override ```std::string DebugPrint(const dp::text::TextMetrics & tm)```, not oprerator<<

Reply:
Removed, it is unused.